dovpanda/core.py

```diff
@@ -90,6 +91,13 @@ def csv_index(res, arguments):
                         f'<code>pd.read_csv({filename}, index_col=0)</code>')
 
 
+@ledger.add_hint('read_csv', 'pre')
+def check_csv_size(arguments):
+    filename = arguments.get('filepath_or_buffer')
+    if getsize(filename) > MAX_CSV_SIZE:
+        ledger.tell('File size is very large. To avoid format issues before the complete file loades, '
+                    f'try:  <code>pd.read_csv({filename}, nrows=5)</code>')
```

Comment:
What happens after I try?
add something like "to check schema is as expected"

Reply:
will be more clear